test/ui-test/testSuites/global_shared/steps/steps.py

```diff
@@ -88,9 +88,9 @@ def step(context, obj):
 ###########################################################################
 
 def the_user_restarts_the_app(context: any):
-    waitFor(lambda: currentApplicationContext().detach(), 500)
+    init_steps.waitFor(lambda: init_steps.currentApplicationContext().detach(), 500)
     time.sleep(5)
-    startApplication(context.userData["aut_name"])
+    init_steps.startApplication(context.userData["aut_name"])
```

Comment:
I think it is not part of the `init_steps` file:
```suggestion
   startApplication(context.userData["aut_name"])
```